scripts/check_samples.py

```diff
@@ -42,7 +42,7 @@ def get_expected_samples(crd_path):
         group = group.split(' ')[1]
         group = strip(group, ".azure.com")
         version = version.split(' ')[1]
-        versionPrefix = re.match('v1(api|alpha1|beta)([0-9][a-z]+)?', version).group() + "*"
+        versionPrefix = re.match('v1(api|alpha1|beta)?([0-9][a-z]+)?', version).group() + "*"
```

Comment:
Why do we need `api|alpha1|beta` to be optional?

Reply:
because the handcrafted MySQL User resource (and other handcrafted resources in the future that don't correspond to an azure API verison) will be called `v1` only, not `v1api`.